config/initializers/constants.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module Orca
-  CAMP_START = Date.parse(ENV.fetch('CAMP_START', '2022-07-23'))
-  CAMP_END = Date.parse(ENV.fetch('CAMP_END', '2022-08-06'))
+  CAMP_START = Date.parse(ENV.fetch('CAMP_START').presence || '2022-07-23')
```

Comment:
Old version would fail if  CAMP_START eq ""

Reply:
ENV.fetch will fail with exception if key does not exists, so the default behavior is broken here

Reply:
```
CAMP_START = Date.parse(ENV['CAMP_START'].presence || '2022-07-23')
```
would fix this